quelpa.el

```diff
@@ -388,7 +394,8 @@ If t, `quelpa' tries to do an upgrade.
     (let ((key (car plist))
           (value (cadr plist)))
       (pcase key
-        (:upgrade (setq quelpa-upgrade-p value))))
+        (:upgrade (setq quelpa-upgrade-p value))
+        (:stable (setq quelpa-stable-p value))))
```

Comment:
Why is a "parsing" function setting a global value again? I wonder what I was thinking when I introduced that again.